Minio/Credentials/MinioEnvironmentProvider.cs

```diff
@@ -19,19 +19,17 @@
 
 namespace Minio.Credentials;
 
-public class MinioEnvironmentProvider : EnvironmentProvider
+public class MinioEnvironmentProvider : IClientProvider
 {
-    public override AccessCredentials GetCredentials()
+    public AccessCredentials GetCredentials()
     {
-        var credentials = new AccessCredentials(GetEnvironmentVariable("MINIO_ACCESS_KEY"),
-            GetEnvironmentVariable("MINIO_SECRET_KEY"), null, default);
+        var credentials = new AccessCredentials(Environment.GetEnvironmentVariable("MINIO_ACCESS_KEY"),
```

Comment:
`MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY` are deprecated.
The newly coming env vars are `MINIO_ROOT_USER` and `MINIO_ROOT_PASSWORD`.
However, we still honor the deprecated ones if the new env vars are not defined.